src/js/utils/math.js

```diff
@@ -0,0 +1,38 @@
+import {OPERATOR, OPERATOR_TEXT} from '../consts/operator.js';
+
+export function calculateDisplayText({prevNumber, operator, nextNumber}) {
+    if (!operator) {
+        return prevNumber;
+    }
+
+    if (!nextNumber && nextNumber != 0) {
+        return `${prevNumber}${OPERATOR_TEXT[operator]}`;
+    }
+
+    return `${prevNumber}${OPERATOR_TEXT[operator]}${nextNumber}`;
+}
+
+const calculateFn = {
+    [OPERATOR.PLUS]: (a, b) => a + b,
+    [OPERATOR.MINUS]: (a, b) => a - b,
+    [OPERATOR.MULTIPLE]: (a, b) => a * b,
+    [OPERATOR.DIVIDE]: (a, b) => Math.floor(a / b),
+};
+
+export function calculateNumber({prevNumber, operator, nextNumber}) {
+    if (!operator) {
+        return prevNumber;
+    }
+
+    if (!nextNumber && nextNumber != 0) {
```

Comment:
`===` or `!==` 으로 비교하시면 더 좋을 것 같습니다 
```suggestion
    if (!nextNumber && nextNumber !== 0) {
```

Reply:
https://github.com/next-step/js-calculator/pull/69/commits/47521c790600b2b65c640bc128403b217efa5152 strict 비교하도록 수정했습니다.